build.assets/Dockerfile-centos7

```diff
@@ -1,5 +1,38 @@
 ARG RUST_VERSION
 
+## GIT2 ###################################################################
+
+# git2 packages are not available on ARM64, so we need to buld it from source.
+FROM centos:7 AS git2
```

Comment:
I wonder if we should just prebuild arm64 git too...

Reply:
I was also thinking about it. I think that the question is how often we are going to update it and how long it takes to build it. In the case of Clang, the answers are "almost never" and "forever". Here, in my opinion, "from time to time" and "quick". Because of the different characteristics, I decided to keep it here, but I don't mind moving it.